### PR TITLE
talks: Add ASEM Expert Group Digitalization slides

### PR DESCRIPTION
**Pull request by razvand**
Takes place online on Friday, December 16, 2022.

### CODE REVIEW

**Review by RazvanN7 — APPROVED** [review]

**Comment by RazvanN7** [comment]
Looks good, but I don't understand the test deployment failure.

**Review by teodutu — APPROVED** [review]
I made some changes so the build succeeds. It was failing because the builder doesn't like it when `.md` files start with `---`. Therefore, I added them to `slides.mdpp` instead. And in order not to display the individual slide files, I changed the name of their folder from `slides/` to `.slides`.

Now the slides won't show up on the website because the builder that embeds them into an iframe is not present on this repo.

**Comment by razvand** [comment]
This doesn't look good. @teodutu, by the team we have the slides rendered on the website, we want to only provide some information on the event. I can fill an `.md` file with that.

I want to:
* Have a direct link to the talk page (similar to the `Events` part), without that ugly `asem-eg-2022-12-16` part.
* Not show the slides as they are. Just show a description (similar to the [Multiplier Event page](https://open-education-hub.github.io/docs/events/m1)); and the slides will be rendered on that description page, once we do the integration. They won't be shown before that happens (in any way).

Could you please create the skeleton for that, and then I will fill the talk description in the required `.md` file.

**Comment by teodutu** [comment]
@razvand done, I think. The skeleton is super minimalistic because I know little about the event itself and about what should be written in this description. Feel free to change it however you see fit.

**Comment by razvand** [comment]
@teodutu, the slide contents are still rendered. And there is a hierarchy of links. See below.

Screenshot from 2023-01-09 10-32-23

**Comment by teodutu** [comment]
What about now?

File: website/docs/talks/asem-eg-2022-12-16.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Logistics, management, evaluation and content should be, as much as possible, already in place.
+
+We stated that technology (computing, Internet, devices) provides the means to get many mundane, repetitive things out of the way and to improve teaching / learning via automation and via global connection (for contributions and collaboration).
+Moreover, technology provides accessibility and fairness to (high quality educational) content to those who request / need it.
+
+We gave on overview of our work so far and the tools and techniques we adapted from the open source world: Markdown format, GitHub, [reveal-md](https://github.com/webpro/reveal-md), Docker, GitLab, CI/CD, scripting & automation.
+
+We summarized the challenges we are currently facing, regarding the effort to generate and convert content, making it easy to configure for personal, integration with existing tools and having the manpower to maintain a large set of repositories.
+
+The talk was well received, with questions / recommendations on having the approach available beyond universities.
```
Comment (teodutu) [inline website/docs/talks/asem-eg-2022-12-16.md]: Maybe give a few examples of points of feedback that you have received and that we are considering?

File: website/docs/talks/asem-eg-2022-12-16.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Logistics, management, evaluation and content should be, as much as possible, already in place.
+
+We stated that technology (computing, Internet, devices) provides the means to get many mundane, repetitive things out of the way and to improve teaching / learning via automation and via global connection (for contributions and collaboration).
+Moreover, technology provides accessibility and fairness to (high quality educational) content to those who request / need it.
+
+We gave on overview of our work so far and the tools and techniques we adapted from the open source world: Markdown format, GitHub, [reveal-md](https://github.com/webpro/reveal-md), Docker, GitLab, CI/CD, scripting & automation.
+
+We summarized the challenges we are currently facing, regarding the effort to generate and convert content, making it easy to configure for personal, integration with existing tools and having the manpower to maintain a large set of repositories.
```
Comment (teodutu) [inline website/docs/talks/asem-eg-2022-12-16.md]: Everything is in the past tense, which makes this look like a log. What about writing "These are the topics we addressed:" at the beginning and then state each point using the present tense?

**Review by teodutu — COMMENTED** [review]
I can't say much about the content of the description itself, only some comments about its form.

File: website/docs/talks/asem-eg-2022-12-16.md (hunk 2 above)
Comment (razvand) [reply website/docs/talks/asem-eg-2022-12-16.md]: Passive voice or past tense?

Comment (teodutu) [reply website/docs/talks/asem-eg-2022-12-16.md]: Yeah, past tense. I was thinking "at least it's not passive voice" and then made a Freudian slip.

Comment (razvand) [reply website/docs/talks/asem-eg-2022-12-16.md]: Will convert to present tense, makes sense.

File: website/docs/talks/asem-eg-2022-12-16.md [hunk 3]
```diff
@@ -5,16 +5,20 @@ As part of the [ASEM Expert Group Digitalisation](https://asem-education.org/pri
...
 
-We summarized the challenges we are currently facing, regarding the effort to generate and convert content, making it easy to configure for personal, integration with existing tools and having the manpower to maintain a large set of repositories.
+We summarize the challenges we are currently facing, regarding the effort to generate and convert content, making it easy to configure for personal, integration with existing tools and having the manpower to maintain a large set of repositories.
 
-The talk was well received, with questions / recommendations on having the approach available beyond universities.
+The talk was well received, with a question whether it is targeted only for universities.
+OpenEdu is focused on education on all levels;
+it provides the methodology, infrastructure and content for teachers and learners alike to deliver and consume educational resources.
+Another question considered the effort in having maintainers for different content repositories;
+we replied that this is a challenge of the project, and we will have to find solutions to ensure maintainers for Open Education Hub repositories.
```
Comment (teodutu) [inline website/docs/talks/asem-eg-2022-12-16.md]:
```suggestion
we replied that this is a challenge of the project, and we will have to find solutions to assign maintainers for Open Education Hub repositories.
```
"ensure" is more commonly used for _things_ and actions.

**Review by teodutu — CHANGES_REQUESTED** [review]
Just a small nit and then it will be fine.

**Review by teodutu — APPROVED** [review]
Both the website and the slides build and render fine locally. This should be OK.

**Pull request merged**